Scripts/RemediationScripts/Remediate-SetStorageAccountMinReqTLSVersion.ps1

```diff
@@ -1,39 +1,39 @@
 ﻿<###
 # Overview:
-    This script is used to set required TLS version for Storage Account in a Subscription.
+    This script is used to set required minimum TLS version for Storage Accounts in a Subscription.
 
 # Control ID:
-    Azure_Storage_DP_Use_Secure_TLS_Version_Trial
+    Azure_Storage_DP_Use_Secure_TLS_Version
 
 # Display Name:
-    Use Approved TLS Version in Storage Account.
+    Use Approved minimum TLS version in Storage Account.
 
 # Prerequisites:
     1. Contributor or higher privileges on the Storage Accounts in a Subscription.
     2. Must be connected to Azure with an authenticated account.
 
 # Steps performed by the script:
     To remediate:
-        1. Validate and install the modules required to run the script.
-        2. Get the list of Storage Accounts in a Subscription that do not use the required TLS version
+        1. Validate and install the modules required to run the script and validating the user..
```

Comment:
Validate and install the modules required to run the script and validating the user.. -> Validate and install the modules required to run the script and **validate the user.**